examples/custom-resources/jwks/nginx-config.yaml

```diff
@@ -0,0 +1,7 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nginx-config
+  namespace: nginx-ingress
+data:
+  resolver-addresses: "kube-dns.kube-system.svc.cluster.local"
```

Comment:
should we put this in < > and callout that they should match to this production environment?

`data:
  resolver-addresses: "<replace_with_your_address"`
`